projects/angular/src/emphasis/alert/alert.ts

```diff
@@ -60,11 +61,11 @@ export class ClrAlert implements OnInit, OnDestroy {
   @Output('clrAlertClosedChange') _closedChanged: EventEmitter<boolean> = new EventEmitter<boolean>(false);
 
   @Input('clrAlertType')
-  set alertType(val: string) {
+  set alertType(val: AlertType) {
```

Comment:
I believe that's a huge breaking change that people are going to complain about right away.